sdk/core/Azure.Core/src/Shared/OperationInternal.cs

```diff
@@ -64,12 +64,12 @@ internal class OperationInternal
         /// <param name="scopeAttributes">The attributes to use during diagnostic scope creation.</param>
         public OperationInternal(
             ClientDiagnostics clientDiagnostics,
-            IOperation operation,
+            IOperation? operation,
```

Comment:
Why can it be null?

Reply:
`OperationInternal<T>` passes `null` here.

Reply:
I don't like it though. Do we use anywhere that `OperationInternal<T>` is derived from `OperationInternal`?

Reply:
For code sharing mostly.